pykickstart/commands/network.py

```diff
@@ -667,4 +676,9 @@ def parse(self, args):
                     msg = formatErrorMsg(self.lineno, msg=_("Bad format of --bridgeopts, expecting key=value options separated by ','"))
                     raise KickstartValueError(msg)
 
+        if retval.bindto == BIND_TO_MAC:
+            if retval.vlanid and not retval.bondopts:
+                msg = formatErrorMsg(self.lineno, msg=_("--bindto=%s is not supported for this type of device" % BIND_TO_MAC))
```

Comment:
The string substitution here should take place outside the call to _().  That is:
`msg=_("--bindto=%s...") % BIND_TO_MAC`

Reply:
Oops, I did it again.